src/shared/components/button/button.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ButtonHTMLAttributes, ReactNode } from "react";
+
+import * as styles from "./button.css";
+
+type ButtonColor = "primary" | "secondary";
+type ButtonSize = "buy" | "ticketbayGlobal";
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  color?: ButtonColor;
+  size?: ButtonSize;
```

Comment:
type을 분리하지 않고 바로 적용해도 괜찮을것 같아요! ( 개인 의견 )
```suggestion
  color?: "primary" | "secondary";
  size?: "buy" | "ticketbayGlobal";
```
유니온 타입이 많지 않으니 그대로 쓰는게 한 눈에 보기 편할것 같아요!
수정은 안해도 될것 같은데 유진님은 어떤게 더 괜찮은지 궁금하네요!!